Src/ReadExternalPressure.c

```diff
@@ -3,18 +3,64 @@
 #include "cmsis_os.h"
 
 #include "ReadExternalPressure.h"
-
 #include "Data.h"
+#include "main.h"
 
-static int READ_EXTERNAL_PRESSURE_PERIOD = 1000;
+static const int READ_EXTERNAL_PRESSURE_PERIOD = 1000;
+static const uint8_t RESET_COMMAND = 0x1E;
+static const int CMD_SIZE = 1;
+static const int CMD_TIMEOUT = 3000;
+static const uint8_t ADC_512_CONV_CMD = 0x42;
+static const uint8_t ADC_READ_CMD = 0x00;
+static uint8_t dataIn;
+static int temp;
```

Comment:
We should use a local variable for this instead, no reason for it to be a global. And we should pick a better name for it.